src/components/ListItem.jsx

```diff
@@ -35,6 +43,10 @@ export function ListItem({ name, data, listToken }) {
 				<label htmlFor="listItem" className="ListItem-label">
 					{name}
 				</label>
+				<button type="submit" onClick={handleDeleteItem}>
```

Comment:
I wonder if this button should be `type="button"` since it's not connected to a form. But then again, we performing a kind of form-like request. Just something to think about.

Reply:
Copy that! Changed the button type